srcpkgs/python3-mpdnotify/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'python3-mpdnotify'
+pkgname=python3-mpdnotify
+version=1.0
+revision=1
+wrksrc=mpdnotify-${version}
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-Pillow python3-mpd2"
+depends="libnotify python3-mpd2 python3-Pillow"
```

Comment:
also python3-setuptools

Reply:
Why do I need setuptools as a runtime dependency?

Reply:
Well, it won't run without, you can check within xbps-src chroot.

I am not sure why setuptools started to inject inself into all it builds, maybe this should be prevented.